mobile_verifier/src/rewarder.rs

```diff
@@ -218,6 +219,11 @@ where
                 tracing::info!("No speedtests found past reward period");
                 return Ok(false);
             }
+
+            if check_for_unprocessed_data_sets(&self.pool, reward_period.end).await? {
```

Comment:
This call would benefit from using the qualified name.

Reply:
as in, keep the `db::` prefix? How qualified are we talking?

Reply:
Hmm. Let's see. 
If there was a `db::` prefixed call here, I would think it was local `rewarder::db::`.

I think my confusion is more around `data_sets`. Nothing about it hints that the rewarder is checking something related to hex boosting. 

I would have a similar confusion if I was looking at the database and saw a plain `data_sets` table.